pkg/platform/kube/functionres/lazy.go

```diff
@@ -1271,6 +1271,26 @@ func (lc *lazyClient) getFunctionVolumeAndMounts(function *nuclioio.NuclioFuncti
 			continue
 		}
 
+		if configVolume.Volume.FlexVolume != nil && configVolume.Volume.FlexVolume.Driver == "v3io/fuse" {
+
+			// make sure the given sub path matches the needed structure. fix in case it doesn't
+			subPath, subPathExists := configVolume.Volume.FlexVolume.Options["subPath"]
+			if subPathExists && len(subPath) != 0 {
+
+				// insert slash in the beginning in case it wasn't given (example: "my/path" -> "/my/path")
+				if subPath[0] != '/' {
+					subPath = "/" + subPath
```

Comment:
`subPAth = filepath.Abs(subpath)`

Reply:
this will add current working directory if it's not abs, so I'll still use '/' instead

> // Abs returns an absolute representation of path.
> // If the path is not absolute it will be joined with the current
> // working directory to turn it into an absolute path. The absolute
> // path name for a given file is not guaranteed to be unique.
> // Abs calls Clean on the result.
> func Abs(path string) (string, error) {
> 	return abs(path)
> }

Reply:
Correct-o